src/libpriv/rpmostree-core.cxx

```diff
@@ -1940,7 +1941,7 @@ find_locked_packages (RpmOstreeContext *self,
           else
             {
               auto repodata_chksum = rpmostreecxx::get_repodata_chksum_repr(*match);
-              if (!g_str_equal (chksum, repodata_chksum.c_str()))
+              if (pkg.digest != repodata_chksum) /* we're comparing two rust::String here */
```

Comment:
It's going to take me a while to adapt to the C++ world where `==` and `!=` can invoke methods.  It just *looks like* C...